app/views/environment_config_form/environment_config_form_integration.html.erb

```diff
@@ -0,0 +1,122 @@
+<% content_for(
+       :page_title,
+       "#{'Error - ' if (@error || @onboarding_form.errors.any?)}Verify Environment Configuration - GOV.UK"
+   )
+%>
+
+<% if @error %>
```

Comment:
are we doing any validation checks here? we might not need this (same with other views)